src/ostorlab/serve_app/oxo.py

```diff
@@ -177,8 +177,80 @@ def mutate(
             return DeleteScanMutation(result=True)
 
 
+class PublishAgentGroupMutation(graphene.Mutation):
```

Comment:
Create

Reply:
i named it `PublishAgentGroupMutation` to align with RE !

Reply:
Ok.